docs/ref/models/fields.txt

```diff
@@ -1505,7 +1505,7 @@ relation works.
 
         user = models.ForeignKey(
             User,
-            models.SET_NULL,
+            on_delete=models.SET_NULL,
```

Comment:
`on_delete` is a required argument and it can be passed a positionally

https://github.com/django/django/blob/33c365781abbcc1b21a31b31d95d344a174df0d5/django/db/models/fields/related.py#L460